client_test.go

```diff
@@ -72,6 +72,27 @@ func TestLogPanic(t *testing.T) {
 	client.Close()
 }
 
+func TestLogError(t *testing.T) {
+	client := testClient()
+	client.Error(errors.New("logged error"))
```

Comment:
I'm unaware of the most idiomatic/gopher way of doing tests, and the testing requirements of this sdk, but I wonder if we should test all the newly introduced functions to make sure each passes the appropriate level.

Reply:
not sure if that was really critical to test it, but added tests